modules/nf-core/happy/prepy/tests/main.nf.test

```diff
@@ -0,0 +1,109 @@
+nextflow_process {
+
+    name "Test Process HAPPY_PREPY"
+    script "../main.nf"
+    process "HAPPY_PREPY"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "happy"
+    tag "happy/prepy"
+
+    test("homo_sapiens - illumina - vcf - bed") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                        [ id:'test' ], // meta map
+                        file(params.test_data['homo_sapiens']['illumina']['test_test2_paired_mutect2_calls_vcf_gz'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_multi_interval_bed'], checkIfExists: true)
+                        ]
+                input[1] = [
+                        [ id:'test2' ], // meta map
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true)
+                        ]
+                input[2] = [
+                        [ id:'test3' ], // meta map
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_fasta_fai'], checkIfExists: true)
+                        ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.preprocessed_vcf).match("bed_preprocessed_vcf") },
```

Comment:
```suggestion
                { assert snapshot(
                    process.out.preprocessed_vcf[0].collect { it instanceof Map ? it : path(it).linesGzip[-5..-1] }
                ).match("bed_preprocessed_vcf") },
```

This will make sure that only the last 5 lines of the VCF are in the snapshot :p (you'll also need to add this to all other non-stub tests of course